docs/developers/meta/sc-build-reference.md

```diff
@@ -157,8 +151,6 @@ After building the contracts, there are three more operations left to perform, b
 
 Calling `cargo run clean` in the meta crate will run `cargo clean` in all wasm crates and delete the `output` folder.
```

Comment:
Clean is also available using: `sc-meta all clean`

Reply:
Fixed.